product_vision.md

```diff
@@ -0,0 +1,65 @@
+    * Where there is disagreement or inability to comply with an accessibility standard, NVDA maintainers  clearly and respectfully argue the case, providing practical alternative solutions.
+    * NVDA maintainers partispate in discussions with standards groups.
+10. Documentation: Ensure users have the knowledge to fully leverage NVDA's features for productivity in work and everyday life:
+    * NVDA offers comprehensive, user-friendly documentation such as an official user guide free of charge.
+    * User-level support is available via various community-run email lists.
+11. Multimodal: Ensure that NVDA supports various modes of interaction, meeting the diverse needs of its users:
+    * NVDA supports multiple input methods including the keyboard, mouse, touch, and braille input devices.
+    * NVDA delivers output through both speech synthesis and braille displays.
+    * To cater to the diversity of user preferences, NVDA strives to make all its features accessible through all input and output modalities. 
+12. Compatibility: Ensure that NVDA works with a wide range of software and hardware to provide maximum accessibility and to ensure users are not left behind in today's rapidly changing technology:
+    * NVDA is regularly updated to be compatible with the latest versions of Windows and popular third-party applications.
+    * NVDA also maintains compatibility with older versions of Windows and third-party applications that are still officially supported by their respective companies, providing accessibility solutions for users on older systems.
+    * NVDA strives to support all common speech synthesizers and refreshable Braille displays that exist today.
+13. Configurable: Ensure the user can configure NVDA to suit their own particular preferences and the task at hand:
+    * NVDA contains many options for configuring what information is reported and how it is presented.
+    * NvDA allows the user to configure attributes of the output such as  the voice rate and pitch, punctuation, and what Braille standard is used.
+    * NVDA allows the user to create configuration profiles, so that the user can easily switch configurations depending on the particular task they are performing.
+14. Extendable: Ensure that NVDA can be easily enhanced and customized to meet specific user needs and requirements:
+    * NVDA allows the creation and integration of community-created add-ons, providing support for extra speech synthesizers, refreshable Braille displays and more third party applications. 
+    * NVDA allows the user to browse for, download and install available add-ons from within NVDA itself.
+    * NVDA provides freedom for add-ons to find innovative solutions by allowing them broad access to NVDA's internals.
+    * NVDA ensures add-ons do not break by postponing major API changes to once per year, announcing these changes many months ahead of time.
```

Comment:
Should this refer to breakages not changes? We regular make major API changes - e.g. by introducing new features.

```suggestion
    * NVDA ensures add-ons do not break by postponing API breakages to once per year, announcing these changes many months ahead of time.
```

Reply:
I agree with @seanbudd